R/rules-spaces.R

```diff
@@ -141,19 +141,29 @@ remove_space_before_opening_paren <- function(pd_flat) {
 }
 
 remove_space_after_opening_paren <- function(pd_flat) {
-  paren_after <- pd_flat$token %in% c("'('", "'['", "LBB")
+  braces <- c("'('", "'['", "LBB")
```

Comment:
Sorry I did not see that before but you are modifying line breaks in a rule for the scope `spaces`. I don't think that function is the right one to implement that functionality. Mostly since it will break styling for the scope `spaces` and also for readability, this is not optimal. Can you put that rule in a related rule in `R/rules-line-breaks.R` or create a new transformer? The latter makes {styler} slower so it's the fallback if we can't accommodate the rule in an existing transformer.

Reply:
Thanks. Makes sense. 

I have created two new transformers now, since I don't see any other transformer that easily incorporate these changes.

Reply:
The performance impact due to new transformers also doesn't look that dire.